binarytree/__init__.py

```diff
@@ -315,7 +314,7 @@ def _get_tree_properties(root):
     }
 
 
-def _get_parent_node(root, node):
+def get_parent_node(root, node):
```

Comment:
The properties in `Node` class is removed and the function `get_parent_node` has been changed into a public function